### ISSUE TITLE
Replace with clipboard: exception

### ISSUE THREAD

**Issue by damithc** [PowerPointLabs/PowerPointLabs#1545]
Win 10, PPT 2013, PL Version 4.1.1.2
Steps: 
* copy screenshot
* select picture in slide
* choose 'replace with clipboard'
```
ShapeRange (unknown member) : Object does not exist.
System.Runtime.InteropServices.COMException
ShapeRange (unknown member) : Object does not exist.
Stack Trace:
   at Microsoft.Office.Interop.PowerPoint.ShapeRange.Copy()
   at PowerPointLabs.ActionFramework.PasteLab.PasteLabActionHandler.ExecuteAction(String ribbonId)
   at PowerPointLabs.ActionFramework.Common.Interface.ActionHandler.Execute(String ribbonId)
```
Note: the picture in the clipboard was bigger than the slide.

**Comment by damithc** [comment]
Got the same error when the clipboard object is not bigger than the slide.

**Comment by damithc** [comment]
Continuing to face this issue.

**Comment by IanTeo** [comment]
@damithc I took a look at the problem and it seems like this was a problem we previously attempted to fix but were unsuccessful.. I have a workaround that you could use in the meantime though.

Instead of directly using Replace With Clipboard, try doing this:
1. Paste the screenshot onto the slide
2. Cut the screenshot (ctrl-x)
3. Replace With Clipboard

The error has something to do with taking images from outside of PowerPoint

**Comment by initialshl** [comment]
I have fixed it in #1546 , probably missed out the solution when debugging in the past.
@IanTeo Can you help me test so that we can merge it in?
